client/app/components/ParameterValueInput.jsx

```diff
@@ -0,0 +1,149 @@
+import { isNull, isUndefined } from 'lodash';
+import React from 'react';
+import PropTypes from 'prop-types';
+import { react2angular } from 'react2angular';
+import { DateInput } from './DateInput.jsx';
+import { DateRangeInput } from './DateRangeInput.jsx';
+import { DateTimeInput } from './DateTimeInput.jsx';
+import { DateTimeRangeInput } from './DateTimeRangeInput.jsx';
+import { QueryBasedParameterInput } from './QueryBasedParameterInput.jsx';
+
+export function ParameterValueInput({
+  type,
+  value,
+  onSelect,
+  enumOptions,
+  queryId,
+  clientConfig, // eslint-disable-line react/prop-types
+  Query, // eslint-disable-line react/prop-types
```

Comment:
Why disable the eslint rule?

Reply:
Alternative is to add this to `propTypes` with type `any`, which also issues another warning. One day we will replace that `angular` services with something more `react`-way, so at this moment, IMHO, it's okay to just take that services as-is and ignore warnings from eslint.